src/components/learner-credit-management/AssignmentEnrollByDateHeader.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  Stack,
+  Icon,
+  IconButtonWithTooltip,
+} from '@openedx/paragon';
+import { InfoOutline } from '@openedx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+const AssignmentEnrollByDateHeader = () => {
+  const intl = useIntl();
+  const internationalizedToolTipContent = intl.formatMessage({
+    id: 'lcm.budget.detail.page.assignments.table.columns.enroll.by.date.tool.tip.content',
+    defaultMessage: 'Failure to enroll by midnight of enrollment deadline date will release funds back into the budget',
+    description: 'On hover tool tip message for the enroll-by date column',
+  });
+  return (
+    <Stack gap={0} direction="horizontal">
+      <span>
+        <p data-testid="assignments-table-enroll-by-column-header" className="mb-0 mr-1">
```

Comment:
This is putting a block element (`p`) in an inline element (`span`), which is not valid HTML. For example, from this w3schools [article](https://www.w3schools.com/html/html_blocks.asp) about block vs. inline elements:

<img width="503" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/f9361ad8-af37-41ae-85d3-1d6a20a925ff">

It's also not really using `Stack` as intended since you have `gap={0}` and are relying on the `mr-1` class name to add the spacing between the text and the icon button (i.e., `Stack` intends to remove the need for using spacing utility classes to implement gaps between elements).

I would suggest changing this to be more inline with the following [Paragon Playground example](https://paragon-openedx.netlify.app/playground/?code=N4Igxg9gJgpiBcIA8BlALgQzAawAQHMMAHAXmAEYBfXKASwCcYw1aIA7EgHRAAsJ7aAL3aYANtwB8nNrlxIAzkQwyoGTAFo0MeSyhcQGefNr42AWxhs08zRgBGomOsv0Io0ersBPdZFEBXMzZ1HhgMWHpJaVlZAFE2V3dPLxo1GGi5AHpFZSkZOQBJSDYAIX80NHYAdVo0HgAVCDcWIgzZSubaIgBhEUs0fQAxDFoAxlxK3Bc3UVxvXDNaKDYTHjRcCAAzKYSZiysaMKhRWjYYVK1cAHdR2cZHQ3PN-zYoeTmsPFPJuvO7fyg%2BBgaG4bQmTVELQACqIsDB9gNuAJ8GtQflZPJ6GAyAU2JsIAB5conM6UMG0YoAQXkOOKZPRuGMghg%2BlOJPSIDBqg0Wh0S300yS3nU3KcHUhXTRMVwmTySEy6E%2BEhAABoQL8LPIEABtEAwKAADQAdPx8CAALpqm5QOpa%2BDagCcDoATObKEA).

```jsx
<Stack gap={1} direction="horizontal">
  <span data-testid="assignments-table-enroll-by-column-header">
    Enroll-by date
  </span>
  <IconButtonWithTooltip
    tooltipContent="Failure to enroll by midnight of enrollment deadline date will release funds back into the budget"
    tooltipPlacement="right"
    src={InfoOutline}
    iconAs={Icon}
    size="inline"
    data-testid="enroll-by-date-tooltip"
    />
</Stack>
```

Reply:
Updated 👍🏽 
There are currently 2 other components that follow the original pattern I implemented against. I am wondering if its worth updating those as well? It would not be considered a big lift.
[MemberStatustableColumnHeader](https://github.com/openedx/frontend-app-admin-portal/blob/e9b406f86a863e332c906100d547a133061ba481/src/components/learner-credit-management/members-tab/MemberStatusTableColumnHeader.jsx#L10)
[MemberEnrollmentsTableColumnHeader](https://github.com/openedx/frontend-app-admin-portal/blob/e9b406f86a863e332c906100d547a133061ba481/src/components/learner-credit-management/members-tab/MemberEnrollmentsTableColumnHeader.jsx#L10)